metadata.yaml

```diff
@@ -24,6 +24,26 @@ missions:
   description: The Health Check mission demonstrates the health check pattern. When you deploy an application, its important to know if it is available and if it can start handling incoming requests. Implementing the health check pattern allows you to monitor the health of an application, which includes if an application is available and whether it is able to service requests.
   metadata:
     level: foundational
+- id: istio-circuit-breaker
+  name: Istio - Circuit Breaker
+  description: The Istio Circuit Breaker mission demonstrates limiting access to a degraded service until it becomes available to handle requests. This helps prevent cascading failure in other services that depend on the failed services for functionality.
+  metadata:
+    level: expert
+- id: istio-distributed-tracing
+  name: Istio - Distributed Tracing
+  description: The Istio Distributed Tracing mission demonstrates how to integrate tracing into services running on Istio.
+  metadata:
+    level: expert
+- id: istio-mutual-tls-acl
+  name: Istio - Mutual TLS and ACL
```

Comment:
FYI: According to the recent discussion on Slack, the Spring Boot team are considering renaming this mission to _Istio Security_.